src/main/resources/assets/structurize/lang/en_us.json

```diff
@@ -675,13 +675,13 @@
   "block.structurize.one_crossed_lr_acacia_oak_timber_frame": "Left Right Crossed Acacia Oak Timber Frame",
   "block.structurize.plain_acacia_dark_oak_timber_frame": "Plain Acacia Dark Oak Timber Frame",
   "block.structurize.side_framed_cactus_stone_timber_frame": "Side Framed Cactus Stone Timber Frame",
-  "block.structurize.placeholderblock": "Placeholder",
+  "block.structurize.placeholderblock": "Placeholder Block",
   "block.structurize.double_crossed_cactus_cobble_stone_timber_frame": "Double Crossed Cactus Cobblestone Timber Frame",
   "block.structurize.double_crossed_cactus_birch_timber_frame": "Double Crossed Cactus Birch Timber Frame",
   "block.structurize.one_crossed_lr_cactus_spruce_timber_frame": "Left Right Crossed Cactus Spruce Timber Frame",
   "block.structurize.double_crossed_spruce_paper_timber_frame": "Double Crossed Spruce Paper Timber Frame",
   "block.structurize.double_crossed_jungle_oak_timber_frame": "Double Crossed Jungle Oak Timber Frame",
-  "item.structurize.sceptertag": "Tag Tool",
+  "item.structurize.sceptertag": "Tagging Tool",
```

Comment:
hmm, not sure but ok

Reply:
Tag tool is better, we got Build Tool, Guard Tool etc in minecolonies

Reply:
I'll change it

Reply:
Wait, isn't it called Build*ing* tool?

Reply:
yes,   `"item.structurize.sceptergold": "Building Tool",`

Reply:
ye but thats because it doesnt mean to Build as action like I am building X. But it means the Building xD

Reply:
Then do ctrl-f "build" and make it consistent 👍

Reply:
k